src/reanimated2/UpdateProps.js

```diff
@@ -23,8 +23,13 @@ const colorProps = [
 
 const ColorProperties = makeShareable(colorProps);
 
-export default function updateProps(viewTag, updates, maybeViewRef) {
+export default function updateProps(viewTag, viewName, updates, maybeViewRef) {
   'worklet';
+  if (viewName && viewName.value) {
```

Comment:
under what condition the viewName will not be set? please at least add a comment here. IMO we should not have a fallback but rather throw but I don't know in what situations the name is not there

Reply:
This fails and takes that default value if `_getViewName` doesn't find corresponding view name. That shouldn't happen for react native and svg components. But I thought someone could try to create animated components out of some other external libraries(like svg) which in fact could do something what svg did which is putting component data in some custom place in a component's object(the svg's `root` case).

We could throw here - it's just a matter of how we want to handle such situation. On the other hand we could also fire up some warning like `unsupported component type` and alert the user that some props may not be set properly.